docs/sphinx/user_guide/using_raja.rst

```diff
@@ -21,16 +21,19 @@ CMake Configuration File
 
 As part of the RAJA installation, we provide a ``RAJA-config.cmake`` file. If
 your application uses CMake, this can be used with CMake's find_package
-capability to import RAJA into your CMake project.
+capability to import RAJA into your CMake project. RAJA also relies on the
+``camp`` library. This is bundled with the RAJA repository, and will install a
+``camp-config.cmake`` file that also works with CMake's find_package.
 
-To use the configuration file, you can add the following command to your CMake
+To use the configuration files, you can add the following command to your CMake
 project::
 
+  find_package(camp)
```

Comment:
Does the user have to do this one? The rest looks great but unless they're using camp themselves this should be optional right?

Reply:
Actually, if it installed correctly, why isn't RAJA's installed cmake file finding it? Are we missing a path we need to store in the FindRAJA.cmake?